config/application.rb

```diff
@@ -12,6 +12,8 @@
 require "sprockets/railtie"
 # require "rails/test_unit/railtie"
 
+require_relative "../lib/verify"
```

Comment:
Will Rails not autorequire everything in `lib`?

Reply:
nope, not since Rails 5, it's discouraged for thread-safety issues - https://blog.bigbinary.com/2016/08/29/rails-5-disables-autoloading-after-booting-the-app-in-production.html